stdlib/public/runtime/ImageInspectionELF.cpp

```diff
@@ -72,9 +72,14 @@ static SectionInfo getSectionInfo(const char *imageName,
   SectionInfo sectionInfo = { 0, nullptr };
   void *handle = dlopen(imageName, RTLD_LAZY | RTLD_NOLOAD);
   if (!handle) {
-#ifdef __ANDROID__
+#if defined(__ANDROID__) || defined(__FreeBSD__)
+#if defined(__ANDROID__)
+    const char *systemPath = "/system/lib";
```

Comment:
Any chance you could change this to “/system” while you’re in the neighbourhood as there are libraries that are blocked on some devices in “/system/vendor” as well as “/system/lib". Non of these libraries will have Swift implementations to worry about. Saves me a PR.

Reply:
:-/ Let's not mix FreeBSD and Android changes.

Reply:
:-/